src/txmempool.cpp

```diff
@@ -1127,7 +1126,7 @@ bool CTxMemPool::_LoadMempool(std::function<bool(const CTransactionRef&, int64_t
     int64_t failed = 0;
     int64_t already_there = 0;
     int64_t unbroadcast = 0;
-    int64_t nNow = GetTime();
+    auto now = NodeClock::now();
```

Comment:
Any reason not to use a steady clock here since it's an interval?

Reply:
Closing, feel free to reopen if not resolved!